packages/client/src/rpc/modules/debug.ts

```diff
@@ -278,4 +288,80 @@ export class Debug {
       message: err.message.toString(),
     }
   }
+
+  /**
+   * Returns a limited set of storage keys belonging to an account.
+   * @param params An array of 5 parameters:
+   *    1. The hash of the block that contains the requested account storage.
+   *    2. The number index of the requested transaction within the block.
+   *    3. The address of the account.
+   *    4. The smallest (hashed) storage key that will be returned. To include the entire range, pass '0x00'.
+   *    5. The maximum number of storage values that will be returned.
+   * @returns A {@link StorageRange} object that will contain at most `limit` entries in its `storage` field.
+   * The object will also contain `nextKey`, the next (hashed) storage key after the range included in `storage`.
+   */
+  async storageRangeAt(params: [string, number, string, string, number]) {
```

Comment:
```suggestion
  async storageRangeAt(params: [string, string, string, string, string]) {
```
most like it will come as a hex encoded number ... not sure about that but do check and update

Reply:
The reference implementation does use `int` for both of these.

https://github.com/ethereum/go-ethereum/blob/4e9775668efc329cd4d12644f5310c1b5ac2ab41/eth/api_debug.go#L212